MsCorePkg/MsCorePkg.ci.yaml

```diff
@@ -104,7 +104,9 @@
             "POLICYREAD",
             "POLICYWRITE",
             "SQRTUNSIGNED",
-            "VARPOL"
+            "VARPOL",
+            "SNP's",

```

Comment:
Why do we have new entries after removing code?

Reply:
MsCorePkg\MacAddressEmulationDxe\MacAddressEmulationDxe.c has the offending word.  Change in spellcheck, I assume.